src/components/icons/EyeIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { createIcon } from '@chakra-ui/react';
+
+export const EyeIcon = createIcon({
+  displayName: 'Eye',
+  viewBox: '0 0 24 24',
+  path: (
+    <>
+      <title>Eye icon</title>
+      <path
+        d="M0.16 8.39C0.29 8.7 3.47 16 10 16C16.53 16 19.71 8.7 19.84 8.39L20 8L19.84 7.61C19.71 7.3 16.53 0 10 0C3.47 0 0.29 7.3 0.16 7.61L0 8L0.16 8.39ZM10 14C5.55 14 2.88 9.39 2.18 8C2.88 6.61 5.55 2 10 2C14.45 2 17.12 6.61 17.82 8C17.12 9.39 14.45 14 10 14ZM7.77772 11.3259C8.43552 11.7654 9.20888 12 10 12C11.0609 12 12.0783 11.5786 12.8284 10.8284C13.5786 10.0783 14 9.06087 14 8C14 7.20888 13.7654 6.43552 13.3259 5.77772C12.8864 5.11993 12.2616 4.60723 11.5307 4.30448C10.7998 4.00173 9.99557 3.92252 9.21964 4.07686C8.44372 4.2312 7.73098 4.61216 7.17157 5.17157C6.61216 5.73098 6.2312 6.44372 6.07686 7.21964C5.92252 7.99556 6.00173 8.79983 6.30448 9.53074C6.60723 10.2616 7.11993 10.8864 7.77772 11.3259ZM8.88886 6.33706C9.21776 6.1173 9.60444 6 10 6C10.5304 6 11.0391 6.21072 11.4142 6.58579C11.7893 6.96086 12 7.46957 12 8C12 8.39557 11.8827 8.78224 11.6629 9.11114C11.4432 9.44004 11.1308 9.69639 10.7654 9.84776C10.3999 9.99914 9.99778 10.0387 9.60982 9.96157C9.22186 9.8844 8.86549 9.69392 8.58579 9.41422C8.30608 9.13451 8.1156 8.77814 8.03843 8.39018C7.96126 8.00222 8.00087 7.60009 8.15224 7.23464C8.30362 6.86918 8.55996 6.55683 8.88886 6.33706Z"
```

Comment:
I think you can optimize it 
```suggestion
        d="M2.16 12.39C2.29 12.7 5.47 20 12 20c6.53 0 9.71-7.3 9.84-7.61L22 12l-.16-.39C21.71 11.3 18.53 4 12 4c-6.53 0-9.71 7.3-9.84 7.61L2 12l.16.39ZM12 18c-4.45 0-7.12-4.61-7.82-6 .7-1.39 3.37-6 7.82-6s7.12 4.61 7.82 6c-.7 1.39-3.37 6-7.82 6Zm-2.222-2.674a4 4 0 1 0 4.444-6.652 4 4 0 0 0-4.444 6.652Zm1.11-4.989a2 2 0 1 1 2.223 3.325 2 2 0 0 1-2.222-3.325Z"
```